Add support for adding descriptions to enum values (#2106)
strawberry-graphql/strawberry: RELEASE.md

Release type: minor

> Add the support for adding descriptions to enum values.

Reviewer: ```suggestion
This release adds support for adding descriptions to enum values.
```

### Example

> Here is an example for adding a description for an enum value.

Reviewer: ```suggestion
```
